src/components/FilterTagsRender.js

```diff
@@ -0,0 +1,37 @@
+import { useHistory, useLocation } from "react-router-dom";
+import { stringify, parse } from "../utilities/queryString";
+import { toggleValueInArray } from "../utilities/filters/toggleValueInArray";
+import FilterTag from "./FilterTag";
+
+export default function FilterTagsRender({ filterPage, filterTags }) {
```

Comment:
I wouldn't append `Render` to the name of the component. I'm sure you can find a better name